.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: Rust CI
+
+on:
+  push:
+    branches: [ "*" ]
+  pull_request:
+    branches: [ "*" ]
+
+jobs:
+  build:
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
it would be better to bind this to a specific version than let it drift and change with "latest" which might break unexpectedly as "latest" moves forward. I'd suggest explicitly binding to the current version only eg: `ubuntu-22.04`

Reply:
Done